### ISSUE TITLE
error when trying to import with as module via rollup

### ISSUE THREAD

**Issue by nylki** [nolanlawson/promise-worker#20]
Hey there,
First of all, I really love the work you have done here and at pouchdb, @nolanlawson :)

I've used `promise-worker` before but wanted to include it this time with `rollup` via `rollup-plugin-commonjs` and `rollup-plugin-node-resolve`.
However I am getting a `ReferenceError: process is not defined` error. 

I suppose this could be a problem with rollup, but as you are using rollup yourself for bundling I thought about asking here if anybody got an idea what the cause might be.

my `rollup.config.js` looks like: 

```.js
// import babel from 'rollup-plugin-babel';
import commonjs from 'rollup-plugin-commonjs';
import nodeResolve from 'rollup-plugin-node-resolve';

export default {
  format: 'iife',
  plugins: [
    nodeResolve({
      jsnext: true,
      main: true
    }),
    commonjs({
        // non-CommonJS modules will be ignored, but you can also
        // specifically include/exclude files
        include: 'node_modules/**',  // Default: undefined

        // if true then uses of `global` won't be dealt with by this plugin
        ignoreGlobal: false,  // Default: false

        // if false then skip sourceMap generation for CommonJS modules
        sourceMap: false  // Default: true
      })
    ]
};
```

and in my index.js i am only doing:

```.js
import PromiseWorker from 'promise-worker';
```

The code is in [this repo](https://github.com/nylki/flowergrow) if you want to try it out yourself.

**Comment by nolanlawson** [comment]
The solution to this problem is to use https://github.com/calvinmetcalf/rollup-plugin-node-globals because otherwise rollup doesn't know what to do with `process`. Let me know if that helps!

**Comment by nylki** [comment]
@nolanlawson Thank you for your response! I actually must have accidentally submitted the issue here. When I was finished writing the issue text, I realized it is not related to promise-worker after all and found out about `rollup-plugin-node-globals` myself, so I thought I did not post the issue here. But my brain must have played tricks on me. -.-

Sorry for stealing your time; but it might proof useful for other users who search for the same thing in the future.
